From ef9f694e4f0c6ea5ae7df61a3b2ada05103a6fd7 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 22 Jul 2016 18:33:49 +0800
Subject: [PATCH] Remove the channel remaping for imx8dv

In imx8dv, the audio card use TDM mode, so the channel mapping
isn't fit for it.

Upstream-Status: Inappropriate [i.MX8dv specific]

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 src/conf/cards/CS42888.conf | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/src/conf/cards/CS42888.conf b/src/conf/cards/CS42888.conf
index 671a284..85ba1e3 100644
--- a/src/conf/cards/CS42888.conf
+++ b/src/conf/cards/CS42888.conf
@@ -38,10 +38,6 @@ CS42888.pcm.surround40.0 {
 		card $CARD
 	}
 	slave.channels 4
-	ttable.0.0 1
-	ttable.1.2 1
-	ttable.2.1 1
-	ttable.3.3 1
 }
 
 
@@ -60,12 +56,6 @@ CS42888.pcm.surround51.0 {
 		card $CARD
 	}
 	slave.channels 6
-	ttable.0.0 1
-	ttable.1.3 1
-	ttable.2.1 1
-	ttable.3.4 1
-	ttable.4.2 1
-	ttable.5.5 1
 }
 
 <confdir:pcm/surround71.conf>
@@ -81,14 +71,6 @@ CS42888.pcm.surround71.0 {
 		card $CARD
 	}
 	slave.channels 8
-	ttable.0.0 1
-	ttable.1.4 1
-	ttable.2.1 1
-	ttable.3.5 1
-	ttable.4.2 1
-	ttable.5.6 1
-	ttable.6.3 1
-	ttable.7.7 1
 }
 
 # vim: ft=alsaconf
-- 
1.9.1

